### PR TITLE
Feature text utility

### PR DESCRIPTION
**Pull request by mariyasada**
adding text-utilities to component Library.do it review

### CODE REVIEW

**Review by sks-21 — APPROVED** [review]
The code looks nice. Good to merge.

File: stylesheets/main.css [hunk 1]
```diff
@@ -44,6 +45,8 @@
     --light-purple:#e879f9;
     --red-color:#ef4444;
     --medium-green:#1a9b49;
+    --ex-light-red: #fecaca;
+    --ex-light-purple:#efd6f3;
 
 }
```
Comment (sks-21) [inline stylesheets/main.css:51]: Use of variable names as primary colour etc is suggested, so even if you decide to change the colour in future, you do not need to change the names from multiple places.

**Review by sks-21 — CHANGES_REQUESTED** [review]

File: stylesheets/main.css (hunk 1 above)
Comment (mariyasada) [reply stylesheets/main.css:51]: I know that thing but before that class, I was already done with 3 features that's why I continue like this. I will keep in mind this for the next project

**Review by JantuDeb — APPROVED** [review]
It's looks fine, you can merge.

**Pull request merged**